cpp/ql/test/library-tests/ptr_to_member/segfault/exprs.ql

```diff
@@ -1,4 +1,5 @@
 import cpp
 
 from Expr e
+where exists(e.toString())
```

Comment:
Here, a delegating constructor call in an uninstantiated template is not recognized as a `ConstructorCall`. I'm not sure if it's reasonable to expect the extractor to be able to figure that out, or if we need to work around it in the library.

```
template<class T>
struct S {
  T second;

  S(piecewise_construct_t, tuple __second) : S(__second, index()) { // <<<This delegating constructor call right here
  }

  template<unsigned long... uls>
  S(tuple t, index) : second(get<uls>(t)...) {
  }
};
```